src/plugins/data/public/search/aggs/buckets/terms.ts

```diff
@@ -63,10 +63,27 @@ export interface TermsBucketAggDependencies {
   getInternalStartServices: GetInternalStartServicesFn;
 }
 
+export interface AggParamsTerms {
+  field: string;
+  order: 'asc' | 'desc';
+  orderBy: string;
+  orderAgg?: AggConfigSerialized;
+  size?: number;
+  missingBucket?: boolean;
+  missingBucketLabel?: string;
+  otherBucket?: boolean;
+  otherBucketLabel?: string;
+  // advanced
+  exclude?: string;
+  include?: string;
+  json?: string;
+}
+
 export const getTermsBucketAgg = ({ getInternalStartServices }: TermsBucketAggDependencies) =>
   new BucketAggType(
     {
       name: BUCKET_TYPES.TERMS,
+      expressionName: 'aggTerms',
```

Comment:
Every agg type will now need to include this field so we know their corresponding function nam (currently optional, but we can make it required once all functions have been created)